staging/src/kubevirt.io/api/migrations/v1alpha1/types.go

```diff
@@ -52,8 +52,6 @@ type MigrationPolicySpec struct {
 	CompletionTimeoutPerGiB *int64 `json:"completionTimeoutPerGiB,omitempty"`
 	//+optional
 	AllowPostCopy *bool `json:"allowPostCopy,omitempty"`
-	//+optional
-	DisableTLS *bool `json:"disableTLS,omitempty"`
```

Comment:
I think we're stuck with this option. Removing it breaks our forward compatibility guarantees.

We can new add options, but never modify the name of existing objects or remove them without incrementing our api versioning.

Reply:
I'm only removing it from Migration Policy CRD, not from KubevirtCR's MigrationConfiguration.
Do we really need to support this although it was just recently merged?

Reply:
>I'm only removing it from Migration Policy CRD, not from KubevirtCR's MigrationConfiguration.
Do we really need to support this although it was just recently merged?

+1
I believe there is no release yet and therefore it's safe to remove.

Reply:
i see, yep this should be safe then as long as we get it in quickly